Assets/VRTK/Source/SDK/Unity/SDK_UnityControllerTracker.cs

```diff
@@ -2,7 +2,12 @@
 namespace VRTK
 {
     using UnityEngine;
+#if UNITY_2017_2_OR_NEWER
+    using UnityEngine.XR;
+#else
     using UnityEngine.VR;
```

Comment:
Sorry I have no time to test this by pulling down, but this looks suspicious: Either it's not needed and can be removed or it should be `using XYZ = UnityEngine.VR.XYZ;` like done everywhere else.

Reply:
The only change that requires the alias is the XRNode.

The other VR class is TrackingSpace, but that hasn't changed between VR and XR

https://github.com/thestonefox/VRTK/blob/73abc8cb15f33a2a187f506a4f5505dbc3a489d0/Assets/VRTK/Source/SDK/Unity/SDK_UnityControllerTracker.cs#L60-L61

So just including the relevant package is enough for that one.

Reply:
I see. Why do they rename everything but don't rename that one. :/

Reply:
Better still why is `UnityEditorInternal.VR;` still that in 2017.2?!